mmv1/products/compute/ServiceAttachment.yaml

```diff
@@ -156,16 +156,13 @@ properties:
           description: |
             The status of the connection from the consumer forwarding rule to
             this service attachment.
-  - !ruby/object:Api::Type::ResourceRef
+  - !ruby/object:Api::Type::String
     name: targetService
     required: true
     immutable: true
-    resource: 'ForwardingRule'
-    imports: 'selfLink'
+    diff_suppress_func: 'TargetServiceDiffSuppress'
```

Comment:
With the most recent change, could this be `'tpgresource.CompareSelfLinkOrResourceName'` instead? Same below, and it might mean we can remove the custom diff suppress function.

Reply:
Sure, I saw yesterday that some other resources have this.
Soon I'll push a new commit with this change.
Thanks for the tip!